demo/package.json

```diff
@@ -8,7 +8,7 @@
     "@ledgerhq/hw-transport-web-ble": "^4.73.2",
     "@ledgerhq/hw-transport-webhid": "^4.73.2",
     "@ledgerhq/hw-transport-webusb": "^4.73.2",
-    "@ledgerhq/live-common": "^9.0.0-alpha.1",
+    "@ledgerhq/live-common": "file:.yalc/@ledgerhq/live-common",
```

Comment:
:ah:

Reply:
I honestly left it on purpose because, how does this even work? I neer a bump regardless for the demo to work 🤷‍♂

Reply:
demo must always point to a release live-common otherwise it can't build and deploy to netlify.

I think your changes is compatible with `^9.0.0-alpha.1` so i suggest you just rollback to it.

it's ok if it does not yet includes your icon addition, it will appear later when we rebump.

Reply:
Removed the yalced, _demo_ will not have latest icons until it's bumped though